types/eslint-utils/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/eslint": ">=7.2.13",
```

Comment:
You shouldn't need these since they're the current version of the types already on DT. The type publisher will generate the correct `@types` dependencies for you.

Reply:
Aha, thanks for the clarification and sorry for the excess PR spam then! I'd misread the docs.